### PR TITLE
Security update

### LINKED ISSUE #1090: mem 1.1.0 and chownr 1.0.1 vulnerabilities
See https://snyk.io/test/github/cds-snc/vac-benefits-directory

vulnerability in mem 1.1.0, affecting us via:
next@6.1.1 › webpack@3.10.0 › yargs@8.0.2 › os-locale@2.1.0 › mem@1.1.0 

The latest pre-release of next (7.0.0-canary.8 as of Sept 1) uses webpack 4.17.1, which does not use yargs. This might fix this vulnerability (but could break things).

vulnerability in chownr via:
next@6.1.1 › uglifyjs-webpack-plugin@1.1.6 › cacache@10.0.4 › chownr@1.0.1

next 7.0.0-canary.8 does not uglifyjs-webpack-plugin so upgrading to the canary might fix this too.

### LINKED ISSUE #1174: move jest-axe to devDependencies

### LINKED ISSUE #1177: Security Updates
Our aim is to update all package dependencies once per sprint. Responsibility for this task is rotated through all the developers. We have come up with the following more or less sane workflow given the the Javascript ecosystem:

In a new branch
- [x] Prune things no longer needed
- [x] Run yarn upgrade-interactive --latest --ignore-engines to find upgraded packages
- [x] Upgrade patch and minor version changes (green and yellow)
- [x] Run tests
- [x] Commit

For each red (major version) upgrade:

- [x] Upgrade
- [x] Run tests
Examine app locally
- [x] Commit

### PR DESCRIPTION
**Pull request by sastels**
resolves #1177  
resolves #1174 
resolves #1090

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,14 +16,14 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
+            - v2-dependencies-{{ checksum "package.json" }}
```
Comment (sastels) [inline .circleci/config.yml:19]: This is how we get CircleCI to stop using old versions of cache, according to the docs

File: components/noscript.js [hunk 2]
```diff
@@ -31,14 +31,13 @@ class Noscript extends Component {
     const { t } = this.props;
 
     return (
-      <noscript role="complementary">
```
Comment (sastels) [inline components/noscript.js:34]: apparently this was always wrong but axe just started failing on it

File: __tests__/components/noscript_test.js [hunk 3]
```diff
@@ -6,12 +6,6 @@ const { axe, toHaveNoViolations } = require("jest-axe");
 expect.extend(toHaveNoViolations);
 
 describe("Test Noscript component", () => {
-  it("displays a message inside a noscript tag", () => {
-    const noscript = mount(<Noscript t={key => key} />);
-
-    expect(noscript.find("div.copy p").text()).toEqual("noscript");
-  });
-
```
Comment (sastels) [inline __tests__/components/noscript_test.js:14]: had to delete this test because now noscript doesn't render on the client, only on the server

**Review by smcmurtry — APPROVED** [review]
Looks good to me! and our bundle sizes went down 😄

**Pull request merged**